### PR TITLE
Minikube: Cleanup everything to start new machine

### PR DESCRIPTION
**Pull request by iamneha**
Fixes issue: #12588

### CODE REVIEW

File: content/en/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -111,4 +111,9 @@ To install Minikube manually on windows using [Windows Installer](https://docs.m
 
 {{% /capture %}}
 
+## Cleanup everything to start fresh
+If you have installed minikube before and when running `minikube start` is throwing error `machine does not exist` then simply wipe existing configuration files.
```
Comment (daminisatya) [inline content/en/docs/tasks/tools/install-minikube.md]: Not just `machine does not exist` error, I faced another issue where removing the config file helped. 

- `Minikube start` used to get stuck at starting the VM, and when checked the logs, I would only see a timeout error in a never-ending loop for a particular API call (Not sure which one particularly) 

Maybe can we frame this a little generic?

**Comment by iamneha** [comment]
@daminisatya 
Maybe you saw this error 
``` 
$ minikube start                                    
Starting local Kubernetes v1.10.0 cluster...
Starting VM...
E0221 14:59:41.221423   25128 start.go:168] Error starting host: Temporary Error: Error configuring auth on host: OS type not recognized.
````

**Comment by daminisatya** [comment]
No @iamneha , not this. I will try to reproduce the error and give you the logs if possible.

File: content/en/docs/tasks/tools/install-minikube.md (hunk 1 above)
Comment (zacharysarah) [reply content/en/docs/tasks/tools/install-minikube.md]: @iamneha This needs better wording. Consider:
> ## Starting fresh after a previous installation
> 
> If you have previously installed minikube, and run:
> ```sh
> minikube start
> ```
> If this command returns an error:
> ```sh
> machine does not exist
> ```
> You need to wipe the configuration files:

**Comment by iamneha** [comment]
Thanks @zacharysarah
I have updated this PR. Please check

**Pull request merged**